setup.py

```diff
@@ -27,7 +26,7 @@
       packages=[package for package in find_packages()
                 if package.startswith('scigym')],
       zip_safe=False,
-      install_requires=['gym==0.11.0',],
+      install_requires=['gym==0.11.0','pytest'],
```

Comment:
Can you remove `pytest` and add it as `tests_require=['pytest']` below?

Reply:
Done. When I run the tests now I get 17 warnings, all of which are deprecation warnings (which are harmless because of the fact that the Keras version has been specified). 

It is possible to filter these warnings so that they don't appear, as described here:

https://docs.pytest.org/en/latest/warnings.html

I'm not sure whether it is better to leave the warnings explicitly (as apparently suggested by PEP-0565) or to filter them out to be neater. I think I'm leaning towards leaving the warnings there, in case other deprecation warnings come up in the future which we do need to address.